qtpy/QtDBus.py

```diff
@@ -8,18 +8,18 @@
 """Provides QtDBus classes and functions."""
 
 import sys
-from . import PYQT5, PYQT6, PYSIDE2, PYSIDE6, PythonQtError, QtBindingMissingModuleError
+from . import PYQT5, PYQT6, PYSIDE2, PYSIDE6, QtBindingsNotFoundError, QtBindingMissingModuleError, QtModuleNotFoundError
 
-if sys.platform == 'linux':
-    if PYQT5:
-        from PyQt5.QtDBus import *
-    elif PYQT6:
-        from PyQt6.QtDBus import *
-    elif PYSIDE2:
-        raise QtBindingMissingModuleError(name='QtDBus')
-    elif PYSIDE6:
+if PYQT5:
+    from PyQt5.QtDBus import *
+elif PYQT6:
+    from PyQt6.QtDBus import *
+elif PYSIDE2:
+    raise QtBindingMissingModuleError(name='QtDBus')
+elif PYSIDE6:
+    if sys.platform != 'win32':
         from PySide6.QtDBus import *
     else:
-        raise PythonQtError("No Qt bindings could be found")
+        raise QtModuleNotFoundError(name='QtDBus', msg='{name} does not exist on this operating system.')
```

Comment:
Maybe we should use here instead the `QtBindingMissingModuleError`?

Reply:
Like this?

```suggestion
        raise QtBindingMissingModuleError(name='QtDBus', msg='{name} does not exist on this operating system.')
```

The more I look at it, the less I like supplying a custom `msg`. 

Is this worth creating creating a new Error class? Something like `QtModuleNotInOSError`?

Reply:
I was thinking in raising the exception without a custom message. However, if adding the information about the OS is important and the message could be reused in other places then maybe a new error class could be worthy :+1:

Reply:
One of my goals is to always provide an answer to "why isn't this available?", because that info could be in any of six (or more?) sets of documentation (or worse: buried on in a forum somewhere(or even worse: the answer is actually the ABSENCE of an answer)). 

Including this information helps the user, but more importantly it helps documents all these ~~stupid~~specific edge cases for future maintainers.

Reply:
Then I would say that having a specific error class like the one you suggest (`QtModuleNotInOSError`) or something like `QtBindingMissingModuleInOSError` is the way to go for this kind of cases

Reply:
Looks like this was resolved as discussed :+1: